### ISSUE TITLE
Studies re procedural steps

### ISSUE THREAD

**Issue by camerons** [thegooddocsproject/templates#68]
Doc: https://github.com/thegooddocsproject/templates/blob/master/writing-tips.md

Re: Writing procedural steps

> Usability studies show that procedures should not be more than seven primary steps long.

Felicity Brand
13:35 14 Nov
"Studies have shown you can use statistics to prove anything". Yeah, we need to reference some facts here.

Josh Sherwood
13:35 14 Nov
Which usability studies? Provide the evidence that backs up these claims

> Usability studies **show that procedures should** not be more than seven primary steps 

Michelle Mortimer
14:17 14 Nov
I'd probably remove this altogether, and just include the recommendation itself. E.g. 'Use no more than 7 primary steps'

Bosco Ho
14:19 14 Nov
Probably related to insights from the "checklist manifesto" in terms of a workable number of steps in an "emergency"

> Too many sub-steps could suggest that you need to break out some of these steps into a new step section.

Hannah McKenzie
13:35 14 Nov
I find this confusing. Does this mean a separate section altogether?
Michelle Mortimer
14:17 14 Nov
+1

Josh Sherwood
13:38 14 Nov
There's a lot of text here [in Writing procedural steps section], a working example might be handy - maybe this is included in another template?

> particularly if you can include **call outs** to the

Hannah McKenzie
13:35 14 Nov
one word [for "callouts"]

**Comment by shreyas1599** [comment]
@camerons I'm trying to open a PR for this. Couple of thoughts:

> Usability studies show that procedures should not be more than seven primary steps long.

A quick google search gave nothing credible. I suggest we remove it or make the change suggested by Michelle Mortimer.

> Too many sub-steps could suggest that you need to break out some of these steps into a new step section.

There's already a point which says something similar: 
> Aim for no more than four sub-steps in any primary step.

Doesn't it seem redundant?

**Comment by camerons** [comment]
I agree that we shouldn't reference metrics if they don't exist, but my gut feeling is that metric sounds like a likely result of a study. If a credible study exists, it would be good to reference it and include this comment. @jaredmorgs might know more as I think he wrote the original text.

**Comment by camerons** [comment]
Feedback from a tech writer I asked:
> I think it was probably the guidelines from the Information Mapping discipline that I mentioned a while ago: a list should contain 5 +- 2 (so, 3 to 7) items. Here's a reference, but there are sure to be more: https://3di-info.com/can-information-mapping-help-us-write-better-technical-documentation/

**Comment by acpkendo** [comment]
The group felt that providing some guidance in this area is better than not providing it, particularly for personas such as software developers (who may not have any background in formal writing in general). @aidanatwork mentioned having some familiarity with research in this area, and offered to take up this issue in order to assemble some guidance we can consider.

**Comment by aidanatwork** [comment]
I've opened a PR that fixes this issue:
https://github.com/thegooddocsproject/templates/pull/187
